.github/workflows/coverity.yml

```diff
@@ -0,0 +1,40 @@
+name: Coverity
+on:
+  schedule:
+    - cron: '0 5 * * *' # Daily at 05:00 UTC
+
+jobs:
+  coverity:
+    name: "Test Suite"
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: osbuild
+    steps:
+
+      - name: Clone repository
+        uses: actions/checkout@v2
+        with:
+         path: osbuild
+
+      - name: Install Dependencies
+        run: |
+          sudo apt-get install python3-setuptools
```

Comment:
```suggestion
        run: sudo apt-get install python3-setuptools
```

(You can do this in a single line, if you want)

Reply:
I think it makes the code that is run easier to stop amongst them `yaml`. My editor highlights them differently :P